client/src/main/scala/drt/client/components/FlightChartComponent.scala

```diff
@@ -15,24 +15,30 @@ import uk.gov.homeoffice.drt.ports.PaxTypes
 object FlightChartComponent {
   case class Props(flightWithSplits: ApiFlightWithSplits)
 
+  case class State(flightWithSplits: ApiFlightWithSplits, showAllNationalities: Boolean)
```

Comment:
We don't need `flightWithSplits` in the state when we already have it in the props